cli/admin/src/test/java/io/pravega/cli/admin/readerGroup/ParseReaderGroupStreamCommandTest.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.readerGroup;
+
+import io.pravega.cli.admin.AdminCommandState;
+import io.pravega.cli.admin.utils.TestUtils;
+import io.pravega.client.ClientConfig;
+import io.pravega.shared.security.auth.DefaultCredentials;
+import io.pravega.test.common.SecurityConfigDefaults;
+import io.pravega.test.integration.utils.SetupUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.rules.Timeout;
+
+import java.io.File;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static io.pravega.test.integration.utils.TestUtils.pathToConfig;
+
+
+/**
+ *  This test is for testing the segment store cli commands.
```

Comment:
This does not say much. If you want to document this test class, write details that are relevant for a developer to understand the approach you have taken to test it, in case it is something that deserves to be highlighted.